open_schools_platform/user_management/users/tests/views/test_users_exceptions.py

```diff
@@ -0,0 +1,155 @@
+import datetime
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+import pytz
+
+from open_schools_platform.user_management.users.services import create_user, create_token
+
+
+class UserExceptionsTests(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+        self.token_creation_url = "/api/user-management/users/token"
```

Comment:
We can use names of urls there instead of urls. It can be useful if we would change urls in urls.py.
https://docs.djangoproject.com/en/4.0/ref/urlresolvers/